semanticdb/semanticdb/shared/src/main/proto/semanticdb.proto

```diff
@@ -267,6 +268,19 @@ message SymbolInformation {
     CLASS = 13;
     TRAIT = 14;
     INTERFACE = 18;
+
+    // =======================
+    // Protobuf-specific kinds
+    // =======================
+    MESSAGE = 19;
```

Comment:
Looks like they are not properly sorted, should be
```suggestion
    MESSAGE = 22;
```

Reply:
Looks similar to https://github.com/sbt/sbt/issues/6392 Eugene I think complained at some point that sbt installed by coursier causes a lot of issues, so that could be it as well

Reply:
I see the CI errors, will get it fixed